ckanext/depositar_theme/templates/home/index/idx_banner.html

```diff
@@ -0,0 +1,24 @@
+<div class="idx banner">
+	<div class="container">
+		<div class="heading">
+			<div class="slogan">
+				<h2>{{ _('deposit・<wbr>discover・<wbr>reuse') }}</h2>
+			</div>
+			<div class="search">
+				<input type="text" class="input-search" placeholder="搜尋資料集...">
+				<button type="submit" class="btn-search"><i class="fa fa-search"></i></button>
+			</div>
+			<div class="function-btn">
+				<a class="btn create-dataset" href="{{ h.url_for('/dataset/new') }}">{{ _('Create Dataset') }}</a>
```

Comment:
`url_for()` 內應填入 route name，避免固定網址因 routing 變動而失效。其他使用 `url_for()` 處，請一併修正

若需顯示文字，亦可使用 [link_for](https://docs.ckan.org/en/2.9/contributing/frontend/templating.html#link-for)

```suggestion
				{% link_for _('Create Dataset'), named_route='dataset.new', class_='btn create-dataset' %}
```